source/constructs/api/catalog/crud.py

```diff
@@ -935,21 +943,38 @@ def get_export_catalog_data():
-                                                                 models.CatalogColumnLevelClassification.account_id,
-                                                                 models.CatalogColumnLevelClassification.region,
-                                                                 models.CatalogColumnLevelClassification.database_name,
-                                                                 models.CatalogColumnLevelClassification.table_name,
-                                                                 models.CatalogColumnLevelClassification.column_name,
-                                                                 models.CatalogColumnLevelClassification.identifier,
-                                                                 models.CatalogColumnLevelClassification.column_value_example,
-                                                                 models.CatalogTableLevelClassification.label_ids,
-                                                                 models.CatalogDatabaseLevelClassification.label_ids,
-                                                                 models.CatalogColumnLevelClassification.comments).all()
+                                             ).filter(
+        and_(models.CatalogColumnLevelClassification.region == models.CatalogTableLevelClassification.region,
+             models.CatalogColumnLevelClassification.database_type == models.CatalogTableLevelClassification.database_type,
+             models.CatalogColumnLevelClassification.database_name == models.CatalogTableLevelClassification.database_name,
+             models.CatalogColumnLevelClassification.table_name == models.CatalogTableLevelClassification.table_name,
+             models.CatalogColumnLevelClassification.region == models.CatalogDatabaseLevelClassification.region,
+             models.CatalogColumnLevelClassification.database_type == models.CatalogDatabaseLevelClassification.database_type,
+             models.CatalogColumnLevelClassification.database_name == models.CatalogDatabaseLevelClassification.database_name)
+    ).distinct(models.CatalogColumnLevelClassification.database_type,
+               models.CatalogColumnLevelClassification.account_id,
+               models.CatalogColumnLevelClassification.region,
+               models.CatalogColumnLevelClassification.database_name,
+               models.CatalogColumnLevelClassification.table_name,
+               models.CatalogColumnLevelClassification.column_name,
+               models.CatalogColumnLevelClassification.identifier,
+               models.CatalogColumnLevelClassification.column_value_example,
+               models.CatalogTableLevelClassification.label_ids,
+               models.CatalogDatabaseLevelClassification.label_ids,
+               models.CatalogColumnLevelClassification.comments).all()
+
+
+def get_catalog_summay_by_provider_region(region: str):
+    return (get_session().query("database_type"),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are unpacking more than three variables while using a function that returns more than one variable. Using a large number of return values is prone to errors. We recommend that you return a small class or `namedtuple` instance instead.

[Learn more](https://docs.python.org/3/library/typing.html#typing.NamedTuple)